lib/doorkeeper/openid_connect/helpers/controller.rb

```diff
@@ -19,7 +19,6 @@ def oidc_authorization_request?
           controller_path == Doorkeeper::Rails::Routes.mapping[:authorizations][:controllers] &&
             action_name == 'new' &&
             pre_auth.valid? &&
-            pre_auth.client &&
```

Comment:
`client` is checked by `pre_auth.valid?`
https://github.com/doorkeeper-gem/doorkeeper/blob/v5.2.0/lib/doorkeeper/oauth/pre_authorization.rb#L74-L83

Reply:
Sounds good! :+1: 

But I'm confused by the code in `validate_client_id`: https://github.com/doorkeeper-gem/doorkeeper/blob/v5.2.0/lib/doorkeeper/oauth/pre_authorization.rb#L74-L78

Why are you checking `:client_id.nil?` there, i.e. on a symbol?